WordPress/gutenberg
# Data list view: Make filter row, table header, and pagination sticky

## What?
In data list views, try making the filter row, table header, and pagination row sticky.

## Why?
These tools can be useful regardless of scroll position. If you're displaying 100 records per row a lot of scrolling is required to access them.

## Testing Instructions
1. In the Site editor navigate to Pages > Manage all pages
2. Check that the ~filter row~, column headers, and pagination stays in view as you scroll



https://github.com/WordPress/gutenberg/assets/846565/d077a762-0cf6-4bea-b1fa-4ecc3a8db40e
